test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/PartialViewResultExecutorTest.cs

```diff
@@ -340,7 +340,8 @@ private PartialViewResultExecutor GetViewExecutor(DiagnosticSource diagnosticSou
                 new CompositeViewEngine(options),
                 new TempDataDictionaryFactory(new SessionStateTempDataProvider()),
                 diagnosticSource,
-                NullLoggerFactory.Instance);
+                NullLoggerFactory.Instance,
+                new TestModelMetadataProvider());
```

Comment:
Same in the remaining tests. If something fails, use `TestModelMetadataProvider.CreateDefaultProvider()` for the particular test class.